Explorer/Assets/DCL/Passport/PassportController.cs

```diff
@@ -279,15 +307,48 @@ private void SetupPassportModules(Profile profile, PassportSection passportSecti
         private void OnProfilePublished(Profile profile) =>
             SetupPassportModules(profile, PassportSection.OVERVIEW);
 
+        private void OpenPhotosSection()
+        {
+            if (currentSection == PassportSection.PHOTOS)
+                return;
+
+            photoLoadingCts = photoLoadingCts.SafeRestart();
+
+            viewInstance!.OverviewSectionButton.SetSelected(false);
+            viewInstance.BadgesSectionButton.SetSelected(false);
+            viewInstance.PhotosSectionButton.SetSelected(true);
+
+            viewInstance.OverviewSectionPanel.SetActive(false);
+            viewInstance.PhotosSectionPanel.SetActive(true);
+            viewInstance.BadgesSectionPanel.SetActive(false);
+            viewInstance.BadgeInfoModuleView.gameObject.SetActive(false);
+            viewInstance.ViewportSoftMask.enabled = false;
+            viewInstance.MainScroll.content = viewInstance.PhotosSectionPanel.transform as RectTransform;
+            viewInstance.MainScroll.verticalNormalizedPosition = 1;
```

Comment:
very minor but maybe it would be more clear if all this actions of the view were encapsulated in a function inside the view named like OpenPhotosSection so that you could simply call viewInstance.OpenPhotosSection